src/ApolloHooksQuery.re

```diff
@@ -1,14 +1,9 @@
-module type Config = {
-  let query: string;
-  type t;
-  let parse: Js.Json.t => t;
-};
-
-type error = {. "message": string};
+open ApolloHooksTypes;
+type queryError = {. "message": string};
 
-type variant('a) =
```

Comment:
Do we need to rename `variant` into `queryVariant` here? It will then be used as `ApolloHooks.Query.queryVariant`, maybe it is better to avoid duplication with `ApolloHooks.Query.variant`?

Reply:
Not sure why I did this anymore, perhaps to avoid collisions. Not against rolling back if there are no name collisions.